tests/models/compare_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Compare the logprobs of two sequences generated by different models, 
```

Comment:
I would suggest moving this file to `tests/models/utils.py`, since that would make it much more consistent with the other utilities (e.g. core/utils.py, spec_decode/utils.py, lora/utils.py) and will also make it possible to put other utils into this file vs. having a one off for this comparison. While doing that, it would also be good to move this comment into a doc string for `check_logprobs_close`.